src/users/components/organisation-access-permissions/organisation-access-permissions.component.html

```diff
@@ -4,28 +4,32 @@ <h2 class="govuk-heading-l">Case management and access</h2>
     *ngIf="jurisdictionPermissionsForm"
     [formGroup]="jurisdictionPermissionsForm"
   >
-    <lib-gov-checkbox
+    <xuilib-gov-checkbox
       [group]="jurisdictionPermissionsForm"
       [config]="{
         value: 'enableCaseManagement',
         label: 'Manage cases for your organisation',
         name: 'enableCaseManagement',
-        hint: 'This provides the user with access to submat and manage claims.',
+        hint: 'This provides the user with access to submit and manage claims.',
         hiddenLabelContext:
-          'This provides the user with access to submat and manage claims '
+          'This provides the user with access to submit and manage claims'
       }"
     >
-    </lib-gov-checkbox>
+    </xuilib-gov-checkbox>
 
-    <ng-container *ngIf="enableCaseManagement">
+    <ng-container>
       <app-solicitor-profile-content
         *ngIf="hasSolicitorProfile"
       ></app-solicitor-profile-content>
       <!-- Display if OrganisationProfileId LIKE 'OGD_*' -->
       <app-ogd-profile-content *ngIf="hasOgdProfile"></app-ogd-profile-content>
```

Comment:
Should this now be split into the individual OGD profiles?

Reply:
yup, that is left for it's own PR.